tests/jake/default.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, poetry2nix, python3, runCommand }:
+{
+  envWheel = poetry2nix.mkPoetryEnv {
+    python = python3;
+    projectDir = ./.;
+    preferWheels = true;
+  };
+  env = poetry2nix.mkPoetryEnv {
```

Comment:
These won't get evaluated if you don't use them in some way. The other tests have examples of how to do this.

Reply:
good to know, figured they did as i got errors before i added overrides, then they stopped.